pkg/server/commands/write.go

```diff
@@ -137,7 +139,7 @@ func (c *WriteCommand) validateNoDuplicatesAndCorrectSize(deletes []*openfgav1.T
 
 func handleError(err error) error {
 	if errors.Is(err, storage.ErrTransactionalWriteFailed) {
-		return serverErrors.NewInternalError("concurrent write conflict", err)
+		return status.Error(codes.Aborted, "transaction aborted")
```

Comment:
should this be `status.Error(codes.Aborted, err)`? i'd like not use the text coming from the error itself